### PR TITLE
CFY-3678 - setting security config as inputs

### PR DESCRIPTION
**Pull request by noakup**

### CODE REVIEW

File: new/openstack-manager-blueprint.yaml [hunk 1]
```diff
@@ -117,10 +117,16 @@ inputs:
   #############################
   # Security Settings
   #############################
-  username:
+  security_enabled:
+    default: true
```
Comment (noakup) [inline new/openstack-manager-blueprint.yaml]: ask barak

File: new/openstack-manager-blueprint-inputs.yaml [hunk 2]
```diff
@@ -57,7 +57,16 @@
 #############################
 # Security Settings
 #############################
-# Username and password to be used for login, if the simple userstore implementation is used.
+# Cloudify REST security is enabled by default. To disable security, set to false
+#security_enabled: true
```
Comment (noakup) [inline new/openstack-manager-blueprint-inputs.yaml]: same as below

File: new/openstack-manager-blueprint-inputs.yaml [hunk 3]
```diff
@@ -57,7 +57,16 @@
...
+
+# Enabling SSL limits communication with the server to SSL only.
+# NOTE: If enabled, the certificate and private key files must reside in resources/ssl.
+#ssl_enabled: false
+
+# Username and password of the Cloudify administrator.
+# This user will also be included in the simple userstore repostiroty if the
+# simple userstore implementation is used.
 #username: 'admin'
```
Comment (noakup) [inline new/openstack-manager-blueprint-inputs.yaml]: should be admin_username and admin_password

File: new/simple-manager-blueprint-inputs.yaml [hunk 4]
```diff
@@ -22,7 +22,16 @@
 #############################
 # Security Settings
 #############################
-# Username and password to be used for login, if the simple userstore implementation is used.
+# Cloudify REST security is enabled by default. To disable security, set to false
+#security_enabled: true
```
Comment (noakup) [inline new/simple-manager-blueprint-inputs.yaml]: ask barak

File: new/simple-manager-blueprint-inputs.yaml [hunk 5]
```diff
@@ -22,7 +22,16 @@
...
+
+# Enabling SSL limits communication with the server to SSL only.
+# NOTE: If enabled, the certificate and private key files must reside in resources/ssl.
+#ssl_enabled: false
+
+# Username and password of the Cloudify administrator.
+# This user will also be included in the simple userstore repostiroty if the
+# simple userstore implementation is used.
```
Comment (noakup) [inline new/simple-manager-blueprint-inputs.yaml]: should be admin_username and admin_password

File: new/simple-manager-blueprint.yaml [hunk 6]
```diff
@@ -36,10 +36,16 @@ inputs:
   #############################
   # Security Settings
   #############################
-  username:
+  security_enabled:
+    default: true
```
Comment (noakup) [inline new/simple-manager-blueprint.yaml]: ask barak

File: new/openstack-manager-blueprint-inputs.yaml [hunk 7]
```diff
@@ -57,9 +57,18 @@
 #############################
 # Security Settings
 #############################
-# Username and password to be used for login, if the simple userstore implementation is used.
-#username: 'admin'
-#password: 'admin'
+# Cloudify REST security is disabled by default. To enable security, set to true.
```
Comment (nir0s) [inline new/openstack-manager-blueprint-inputs.yaml]: So we decided it's off by default for 3.3?

File: new/openstack-manager-blueprint-inputs.yaml [hunk 8]
```diff
@@ -57,9 +57,18 @@
...
+#security_enabled: false
```
Comment (nir0s) [inline new/openstack-manager-blueprint-inputs.yaml]: we should probably also state above that if security is not enabled, the other inputs are irrelevant..

**Pull request merged**